src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_28 calendar--first-day_fri">
```

Comment:
I've just thought we can write  calendar--month-length--28 or calendar_month-length_28
But if calendar--month-length_28 is possible I like it more